netdata.rst

```diff
@@ -0,0 +1,56 @@
+.. _netdata-section:
+
+=======
+Netdata
+=======
+
+This application installs and configures a Netdata Agent [#NDA]_ instance
```

Comment:
IMO, inline link are much more readable then footnotes.
Also footnotes are not used inside other application pages.